File: 백준/Bronze/4153. 직각삼각형/직각삼각형.cc
#include <iostream>

using namespace std;

int main() {

    while (true) {
        double a, b, c;
        cin >> a >> b >> c;

        if (a == 0 && b == 0 && c == 0) return 0;

        int squaredA = a * a;
        int squaredB = b * b;
        int squaredC = c * c;

        if (squaredC == squaredA + squaredB || squaredB == squaredA + squaredC || squaredA == squaredB + squaredC) {
            cout << "right\n";
        }
        else cout << "wrong\n";
    }

    return 0;
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
  a, b, c = map(int, input().split())
  if (a == 0 and b == 0 and c == 0):
    break
  if (pow(c, 2) == pow(a, 2) + pow(b, 2) or
      pow(b, 2) == pow(a, 2) + pow(c, 2) or
      pow(a, 2) == pow(b, 2) + pow(c, 2)):
    print("right")
  else:
    print("wrong")